e2e-tests/commands/insurance/complete-export-contract-section.js

```diff
@@ -50,6 +52,16 @@ const completeExportContractSection = ({
         fixedSumAmount: agentChargeFixedSumAmount,
         percentageMethod: agentChargeMethodPercentage,
       });
+
+      if (alternativeCurrency) {
+        cy.go('back');
```

Comment:
cy.clickBackButton?

Reply:
We don't have such a command; I think we tried before and causes some strange cypress issues when called within other commands.

Reply:
Updated to use `cy.clickBackLink();`